UnitystationLauncher/Models/Installation.cs

```diff
@@ -46,8 +46,7 @@ public Installation(string folderPath)
             }
             else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
             {
-                var archives = Directory.EnumerateDirectories(path);
-                exe = archives.SingleOrDefault(s => Regex.IsMatch(Path.GetFileName(s), @".*station\.app"));
+                exe = Path.Combine(path, "Unitystation.app", "Contents", "MacOS", "unitystation");
```

Comment:
I made it search for the exe so forks can use a different name from `Unitystation`.
I would either implement the search functionality so it works, fix the file name, or make all of the other platform specific implementations match this one.

Reply:
done